app/dashboard/models.py

```diff
@@ -356,6 +352,16 @@ def fetch_issue_comments(self, save=True):
         return comments
 
 
+class BountyFulfillment(SuperModel):
+    fulfiller_address = models.CharField(max_length=50)
+    fulfiller_email = models.CharField(max_length=255, blank=True)
+    fulfiller_github_username = models.CharField(max_length=255, blank=True)
+    fulfiller_name = models.CharField(max_length=255, blank=True)
+    fulfiller_metadata = JSONField(default={}, blank=True)
+
+    bounty = models.ForeignKey(Bounty, on_delete=models.CASCADE)
```

Comment:
do we want to add a `related_name` back from the Bounty table?

Reply:
Added.